src/foam/java/refinements.js

```diff
@@ -1516,6 +1516,43 @@ foam.CLASS({
 });
 
 
+foam.CLASS({
+  package: 'foam.java',
+  name: 'DateTimeUTCJavaRefinement',
+  refines: 'foam.lang.DateTimeUTC',
+  // flags: ['java'],
+  mixins: [ 'foam.java.JavaCompareImplementor' ],
```

Comment:
Is this necessary? Isn't it already refined into DateTime?

Reply:
yes, removed.